infra/site-cookbooks/cassandra-linux-tweaks/recipes/default.rb

```diff
@@ -7,61 +7,22 @@
 # All rights reserved - Do Not Redistribute
 #
 
-
-bash 'update net.core.rmem_max' do
-  code <<-EOH
-    sysctl net.core.rmem_max=16777216
-  EOH
-end
-
-bash 'update net.core.wmem_max' do
-  code <<-EOH
-    sysctl net.core.wmem_max=16777216
-  EOH
-end
-
-bash 'update net.core.rmem_default' do
-  code <<-EOH
-    sysctl net.core.rmem_default=16777216
-  EOH
-end
-
-bash 'update net.core.wmem_default' do
-  code <<-EOH
-    sysctl net.core.wmem_default=16777216
-  EOH
-end
-
-bash 'update net.core.optmem_max' do
-  code <<-EOH
-    sysctl net.core.optmem_max=40960    
-  EOH
-end
```

Comment:
the commands are not executed independently anymore, but we just call the entire tweaks script (startup script). We do this manually while cheffing, so a new already booted instance is the same as a rebooted instance